### PR TITLE
[Mobile v1] Splash page #2329

### PR DESCRIPTION
**Pull request by ryota-murakami**
**Status**
- [x] Ready for review

**Deploy after merge (delete what needn't be deployed)**
- mobile

**Release notes for users (delete if codebase-only change)**
- [Mobile v1] Splash page #2329

## current progress

- [x] add github button
- [x] add button styled
- [x] fix shown bottom menu without login https://github.com/withspectrum/spectrum/pull/2980 @ArturKlajnerok
- [x] fix emoji color
- ~~[ ] fix auth login not work~~ see: https://github.com/withspectrum/spectrum/pull/2880#issuecomment-385764967 
- [x] add Code of Conduct

### screenshot
screen shot 2018-05-02 at 4 28 55

### CODE REVIEW

**Comment by mxstbr** [comment]
This is getting there! 😻Can we hide the tabs at the bottom if a user isn't signed in by chance?

**Comment by ryota-murakami** [comment]
@mxstbr thx your massage(^_−)−☆
i see, i’ll check later.
when I finished today’s some businesses(Personal relationship etc.).

BTW: I’d like to focus on spectrum project, next twin holidays, so could you tell me your location & time  one?
i’d like to response massage soon if is it possible.

I live in Tokyo, Japan. GMT+9(pretty fast timezone😅)

**Comment by mxstbr** [comment]
> I’d like to focus on spectrum project, next twin holidays, so could you tell me your location & time one?

I'm in Vienna, Austria—Brian and Bryn are in San Franciso, USA! Just ping us in https://spectrum.chat/spectrum/open

**Comment by ArturKlajnerok** [comment]
Hey @mxstbr & @ryota-murakami I've been looking at navigation and I've proposed a changed for users that are not signed here : #2980

**Comment by ryota-murakami** [comment]
@mxstbr i reflect https://github.com/withspectrum/spectrum/pull/2980 by rebase, and update screenshot:green_apple:(rebase is my daily dasis)

so, i'm trying to do  **current progress** paragrah's task.

i have no idea about **fix emoji color**.

i copied emoji svg from web, and apply to mobile it.
but emoji color was not apply correctly.

- web version emoji code
https://codepen.io/ryota-murakami/pen/KRWxxL

- web version emoji code in Expo(`svg`,`g`,`path` were replace Expo provide it)
screen shot 2018-05-01 at 16 56 28


shoud i split problem and open issue about this?
perhaps i need research of `react-native-svg`.

**Comment by mxstbr** [comment]
Don't worry about the SVG for now, we'll have to move all of our icons over eventually anyway—for now, this is fine!

**Comment by ArturKlajnerok** [comment]
@ryota-murakami `react-native-svg` has a slightly different properties then the web SVG. In this case `fill-rule="evenodd"` would be `fillRule="evenodd"`. 

The emoji color fill should be like :
```
<Svg width="64" height="64" viewBox="0 0 32 32">
    <G>
      <Path
        fill={theme.text.alt}
        fillRule="evenodd"
        d="M26,16c0,5.523 -4.477,10 -10,10c-5.523,0 -10,-4.477 -10,-10c0,-5.523 4.477,-10 10,-10c5.523,0 10,4.477 10,10Zm2,0c0,6.627 -5.373,12 -12,12c-6.627,0 -12,-5.373 -12,-12c0,-6.627 5.373,-12 12,-12c6.627,0 12,5.373 12,12Zm-17.5,0c0.829,0 1.5,-0.672 1.5,-1.5c0,-0.828 -0.671,-1.5 -1.5,-1.5c-0.829,0 -1.5,0.672 -1.5,1.5c0,0.828 0.671,1.5 1.5,1.5Zm12.5,-1.5c0,0.828 -0.671,1.5 -1.5,1.5c-0.829,0 -1.5,-0.672 -1.5,-1.5c0,-0.828 0.671,-1.5 1.5,-1.5c0.829,0 1.5,0.672 1.5,1.5Zm-6.999,8c2.45,0 4.534,-1.715 5,-4c0.232,-1.14 -3,-1.5 -5,-1.5c-2,0 -5.259,0.231 -5,1.5c0.466,2.285 2.549,4 5,4Zm2,-2c0,0 -0.896,0.5 -2,0.5c-1.105,0 -2,-0.5 -2,-0.5c0,0 0.895,-1 2,-1c1.104,0 2,1 2,1Z"
      />
    </G>
</Svg>
```

Hope this will work as expected 😃

**Comment by ryota-murakami** [comment]
@ArturKlajnerok thanks your help, emoji color fixed by your snipet, pretty cool!:rocket:

**Comment by ryota-murakami** [comment]
@mxstbr i attempted **fix auth login not work**.
but according to Expo doc, we need client id of auth provider.
https://docs.expo.io/versions/latest/sdk/auth-session#example

i think that is a owner's work, so i droped **fix auth login not work**.

also i finished other works currently, could you please review it?:smile:

File: mobile/components/Login/style.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    height: 51px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    border-radius: 8px;
+    padding: 8px;
+    margin-top: 16px;
+    padding-right: 16px;
+    background-color: ${color};
```
Comment (mxstbr) [inline mobile/components/Login/style.js]: Please move this out of the `SigninButton`—the way this is built right now, the `Container` component will be recreated on every render. That's very expensive!

This is what it should look like:

```JS
const Container = styled.TouchableOpacity`
  background-color: ${props => props.color};
  // ...
`

const SigninButton = ({ color, children, onPress }: SigninButtonProps) => {
  return <Container onPress={onPress} color={color}>{children}</Container>;
}
```

**Review by mxstbr — COMMENTED** [review]
One tiny nitpick, otherwise this looks great code-wise! Thanks so much for digging into this :100:

File: mobile/components/Login/style.js (hunk 1 above)
Comment (ryota-murakami) [reply mobile/components/Login/style.js]: @mxstbr I just fix it!
I'v not noticed this performance issue until now.
thank you for your insightful review:smile:

**Review by mxstbr — APPROVED** [review]
Perfecto, beautiful work. Thanks so much for digging into this

**Pull request merged**